### ISSUE TITLE
Card Tsar retiring mid-turn causes mixed answers

### ISSUE THREAD

**Issue by pidg** [grifferz/pah-irc#44]
When a Card Tsar retires and it's about to announce the choices, it picks a new black card then announces a mixture of old and new choices.

```
21:25 < PAH> The new Card Tsar is paul.
21:26 < PAH> Time for the next Black Card:
21:26 < PAH> → In her latest feature-length film, Tracy Beaker struggles with _____ for the first time.
21:26 < PAH> Now message me your answers please!
21:26 <@Paul> pah retire
21:26 < PAH> paul: Okay, you've been dealt out of the game.
21:26 < PAH> paul: If you want to join in again later then type "PAH: deal me in"
21:26 < PAH> The new Card Tsar is sarah.
21:26 < PAH> Time for the next Black Card:
21:26 < PAH> → What never fails to liven up the party?
21:26 < PAH> Now message me your answers please!
21:27 < PAH> 2 people just played! We're just waiting on daviey to make their play.
21:27 < PAH> All plays are in. No more changes!
21:27 < PAH> sarah: Which is the best play?
21:27 < PAH> ==============================
21:27 < PAH> 1 → What never fails to liven up the party? Sex with Patrick Stewart.
21:27 < PAH> 2 → In her latest feature-length film, Tracy Beaker struggles with teenage pregnancy 
for the first time.
21:27 < PAH> 3 → What never fails to liven up the party? Being a dick to children.
21:27 < PAH> ==============================
21:27 < dg> er
```

**Comment by grifferz** [comment]
It has perhaps not cleared out all the existing plays, so a couple more plays come in and it thinks it has a completed hand but in reality some of the plays were left over from before.
